deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml-review

```diff
@@ -0,0 +1,493 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.0.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Full Lifecycle"
+    categories: "Storage,Cloud Provider"
+    certified: "false"
+    containerImage: stg-artifactory.haifa.ibm.com:5030/ibm-block-csi-operator:0.1.1
+    createdAt: "2019-09-06T16:45:00Z"
+    description: "Run IBM Block CSI Driver on OpenShift."
+    repository: https://github.com/IBM/ibm-block-csi-operator
+    support: IBM
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "IBMBlockCSI",
+          "metadata": {
+            "name": "ibm-block-csi",
+            "namespace": "kube-system"
+          },
+          "spec": {
+            "controller": {
+              "repository": "stg-artifactory.haifa.ibm.com:5030/ibm-block-csi-controller-driver",
+              "tag": "1.0.0_b40_origin.develop"
+            },
+            "node": {
+              "repository": "stg-artifactory.haifa.ibm.com:5030/ibm-block-csi-node-driver",
+              "tag": "1.0.0_b40_origin.develop"
+            }
+          }
+        },
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "Config",
+          "metadata": {
+            "name": "config",
+            "namespace": "kube-system"
+          },
+          "spec": {
+            "defineHost": true,
+            "nodeAgent": {
+              "repository": "stg-artifactory.haifa.ibm.com:5030/ibm-node-agent",
+              "tag": "0.0.1",
+              "port": "10086"
+            }
+          }
+        },
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "NodeInfo",
+          "metadata": {
+            "name": "node-name"
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: "Operator for IBM block storage CSI driver"
+  description: |
+    **IBM Block CSI driver** is a Container Storage Interface (CSI) Driver for IBM block storage systems which enables container orchestrators to manage the life-cycle of persistent storage.
+
+    This is the official operator to deploy and manage IBM block storage CSI driver.
+
+    Supported container platforms:
+      - Openshift v4.1
+      - Kubernetes v1.13
+
+    Supported IBM storage systems:
+      - IBM FlashSystem 9100
+      - IBM Spectrum Virtualize
+      - IBM Storwize
+      - IBM FlashSystem A9000\R
+
+    Supported operating systems:
+      - RHEL 7.x (x86 architecture)
+      - RHCOS 4.1 (x86 architecture)
+
+    ## Pre-requisites
+    
+    ### Worker nodes preparation
+    Perform these steps for each worker node in OpenShift:
+    
+    ### 1. Install Linux packages to ensure Fibre Channel and iSCSI connectivity
+    Skip this step, if the packages are already installed.
+    
+    RHEL 7.x:
+    ```bash
+    $ yum -y install sg3_utils
```

Comment:
the sg3_utils was removed from the driver file. please consult with Ran regarding if it should also be removed from here.

Reply:
remove it.